src/bootstrap/lib/overload-protection.js

```diff
@@ -0,0 +1,32 @@
+const configure = ({
+  production = "production", // if production is false, detailed error messages are exposed to the client
+  clientRetrySecs = 1, // Retry-After header, in seconds (0 to disable) [default 1]
+  sampleInterval = 5, // sample rate, milliseconds [default 5]
+  maxEventLoopDelay = !isNaN(
+    parseInt(process.env.OVERLOAD_PROTECTION_MAX_EVENT_LOOP_DELAY, 10),
```

Comment:
There was a request for being able to override variables c/o environment variables as well. At the moment the only one that is being tweaked is `maxEventLoopDelay`. If others are needed, they can be exposed as well.